docs/examples/basic-requirements.md

```diff
@@ -33,27 +39,26 @@ export PATH=$PATH:~/bin
 mv kubectl kops  ~/bin
 ```
 
-Finally, some of our exercises use the "jq" utility that is available on modern linux distributions. Please ensure to install it too. Some examples of how to do it:
+Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to install it as well. Some examples of how to do it:

```

Comment:
This should be enough for someone to figure out how to install `jq`. The rest of the text below can be removed.
```suggestion
Finally, some of our exercises use the `jq` utility which is available on modern linux distributions. Please ensure to [install](https://stedolan.github.io/jq/download/) it as well.

```